CONTRIBUTING.md

```diff
@@ -80,18 +80,53 @@ CI/CD will run tests, and if successful, do deploys.
 Documentation is built and deployed, and code snippets
 are exported to the service contract repo as a pull request.
 
-Updating and uploading npm packages only happens if the HEAD commit of the main branch
-contains `[release]` in its description or the PR title starts with `feat` or `fix`.
+Updating and uploading npm packages only happens if the HEAD commit of the main branch or a release-* branch
+contain `[release]` and `[bump-version]` in its description and the PR title starts with `feat` or `fix`.
+
+#### Examples
+
+1. If you want to release a release candidate version to npm with version bump when you are commiting to a release-* branch.
+```bash
+  feat(dataParser): add xml support [release][bump-version]  
+  fix(graphUtils): circular path detection [release][bump-version] 
+```
+2. If you want to release a release candidate version to npm without version bump when you are commiting to a release-* branch.
+```bash
+  feat(dataParser): add xml support [release]
+  fix(graphUtils): circular path detection [release]
+```
+
+3. If you want to release a npm verison using master branch
+```bash
+  feat(dataParser): add xml support [release]
+  fix(graphUtils): circular path detection [release]
+```
+
+4. If you want to release a version bumped package from master branch
```

Comment:
Not sure I understand this sentence

Reply:
Sorry, my wording makes no sense here. This is to release patches and minor version updates to the stable release